iati/about/models.py

```diff
@@ -1,8 +1,145 @@
+    subpage_types = ['about.AboutSubPage', 'about.CaseStudyIndexPage', 'about.HistoryPage', 'about.PeoplePage']
+
+
+class AboutSubPage(AbstractContentPage):
+    """A model for generic About subpages."""
+
+    subpage_types = ['about.AboutSubPage']
+
+    multilingual_field_panels = [
+        InlinePanel('about_sub_page_documents', label='About subpage attachments'),
+    ]
+
+
+class AboutSubPageDocument(Orderable):
+    """A model for About sub page documents."""
+
+    page = ParentalKey(AboutSubPage, related_name='about_sub_page_documents')
+    document = models.ForeignKey(
+        'wagtaildocs.Document',
+        on_delete=models.CASCADE,
+        related_name='+'
+    )
+
+    panels = [
+        DocumentChooserPanel('document'),
+    ]
+
+
+class CaseStudyIndexPage(AbstractIndexPage):
+    """A model for the Case Studies Index page."""
+
+    subpage_types = ['about.CaseStudyPage']
+
+    @property
+    def case_studies(self):
+        """Get all CaseStudyPage objects that have been published."""
+        case_studies = CaseStudyPage.objects.child_of(self).live()
+        return case_studies
+
+    def get_context(self, request):
+        """Overwrite the default wagtail get_context function to allow for pagination.
+
+        Use the functions built into the abstract index page class to apply pagination, limiting the results to 3 per page.
+
+        """
+        children = self.case_studies
+        paginated_children = self.paginate(request, children, 3)
+        context = super(CaseStudyIndexPage, self).get_context(request)
+        context['case_studies'] = paginated_children
+        return context
+
+
+class CaseStudyPage(AbstractContentPage):
+    """A model for Case Study pages."""
+
+    parent_page_types = ['about.CaseStudyIndexPage']
+    subpage_types = []
+
+    feed_image = models.ForeignKey(
+        'wagtailimages.Image',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name='+'
+    )
+
+    multilingual_field_panels = [
+        ImageChooserPanel('feed_image'),
+        InlinePanel('case_study_documents', label='Case study attachments'),
+    ]
+
+
+class CaseStudyDocument(Orderable):
+    """A model for Case Study documents."""
+
+    page = ParentalKey(CaseStudyPage, related_name='case_study_documents')
+    document = models.ForeignKey(
+        'wagtaildocs.Document',
+        on_delete=models.CASCADE,
+        related_name='+'
+    )
+    panels = [
+        DocumentChooserPanel('document'),
+    ]
+
+
+class HistoryDateBlock(StreamBlock):
+    """A block for History page info."""
+
+    date_block_editor = StructBlock([
+        ('heading', CharBlock(required=False, max_length=100)),
+        ('description', TextBlock(required=False))
+    ])
+
+
+class HistoryPage(AbstractContentPage):
+    """A model for the History page."""
+
+    subpage_types = []
+
+    date_panel = StreamField(HistoryDateBlock, null=True, blank=True)
```

Comment:
Eventually add help_text that tells users this should be in (reverse) chronological order.